emcc.py

```diff
@@ -1565,6 +1565,10 @@ def get_bitcode_file(input_file):
           options.debug_level = 4
           shared.Settings.DEBUG_LEVEL = 4
 
+      # For asm.js, the generated JavaScript could preserve LLVM value names, which can be useful for debugging.
+      if options.debug_level > 3 and not shared.Settings.WASM:
```

Comment:
I think this should be `>= 3`?

Reply:
For some reason I thought that I wrote >= when I submitted this.

Fixed.